sdk/communication/communication-phone-numbers/src/mappers.ts

```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT License.
 
 import type { SipTrunk } from "./models.js";
-import type { TrunkUpdate as RestSipTrunk } from "./generated/src/siprouting/models/index.js";
+import type { SipTrunk as RestSipTrunk } from "./generated/src/siprouting/models/index.js";
```

Comment:
This will need both models, function "transformFromRestModel" should take response SipTrunk from Rest level and map it to SipTrunk from ./models.js
Function "transformIntoRestModel" should take SipTrunk from ./models.js and map it to TrunkUpdate from Rest level.